.github/workflows/test-actions.yaml

```
@@ -66,6 +66,20 @@ jobs:
       - name: Check metrics file
         run: |
           cat metrics
+  publish-npm-integration-build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Create fake package.json
+        run: |
+          echo '{"name": "test-project", "version": "0.0.1", "scripts": { "build-npm": "mkdir ./dist-types && cp package.json ./dist-types" } }' > ./package.json
+      - name: Test publish-npm-integration-build action
+        uses: qlik-oss/qmfe-actions/publish-npm-integration-build@bgd/int-build-action
```

Will update before merging
```suggestion
        uses: qlik-oss/qmfe-actions/publish-npm-integration-build@v1
```